helm/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nexus-webapp
+  annotations:
+    kubernetes.io/ingress.class: ingress-nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "180"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "180"
+spec:
+  rules:
+  - host: {{ .Values.host }}
+    http:
+      paths:
+      - path: /
```

Comment:
I'd actually remove this by default. The way the root-webpage is setup currently causes the application to return 200 OK for any path requested (the intent was for just /) and it will cause many false-positives for security scans.

Reply:
I'm going to leave the root-webpage route in there for now, because /onearth is currently proxying to a node outside the EKS cluster so I don't want to move the nexus webapp to the root path or nexus might try to pick up requests intended for OnEarth. When OnEarth moves to a k8s deployment, we can revisit this